CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - By default, temporary directory for application building is created in
   `~/.cartridge/tmp`
 - Commands usage messages are prettified
+- `path` argument for `cartridge pack` command isn't required.
```

Comment:
I assume not only `pack` but also `build`

Reply:
This is **changed** section, this behaviour was *changed* only for `pack`. Build command was *added* and it's mentioned in **added** section.